source/slang/syntax.h

```diff
@@ -1166,13 +1157,15 @@ namespace Slang
 
     //
 
-    inline BaseType GetVectorBaseType(VectorExpressionType* vecType) {
-        return vecType->elementType->AsBasicType()->baseType;
+    inline BaseType GetVectorBaseType(VectorExpressionType* vecType)
+    {
```

Comment:
I wonder if this routine ever gets used. It seems dangerous if it does, since it could only apply to fully-specialized code and nothing using generics.

(There's a lot of legacy code lying around in here from the early days of the project, and this PR is helping me notice some of the dusty corners we might have been neglecting).